Dockerfile

```diff
@@ -27,6 +27,8 @@ ADD ./configurator.py /configurator.py
 RUN chmod a+x /configurator.py
 ADD ./renew.sh /usr/local/bin/renew.sh
 RUN chmod a+x /usr/local/bin/renew.sh
+ADD ./config.sh /usr/local/bin/config
+RUN chmod a+x /usr/local/bin/config
```

Comment:
these two steps can be deleted in favor of using the `config` command in the `compat` image

Reply:
this still stands

Reply:
done